src/OpenLoco/Company.cpp

```diff
@@ -15,6 +17,8 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco
 {
+    static loco_global<CompanyId, 0x009C68EB> _updating_company_id;
```

Comment:
We try to keep `loco_global`s to a minimum and constrained to particular files. This should be retrieved from the `GameCommands` namespace instead, using a getter function.

Reply:
This doesn't exist anywhere else in the code for `_updating_company_id`, and I've simply copied the existing use cases for it. I'd call this request scope creep and I don't think it should be my job to refactor the whole `_updating_company_id` global, that should be the job of an entirely separate and dedicated PR.

Reply:
Hmm, I'd rather leave that kind of refactoring to a separate PR. Feel free to pick it up, though.

Reply:
I'm happy to do the refactor but I agree, it should be in another PR